src/helpers/convert-gql.js

```diff
@@ -16,14 +16,37 @@ const getGraphQLType = (value) => {
       return 'String'
     }
 
-    if (!isNaN(Date.parse(value))) {
+    if (isValidDate(value)) {
       return 'DateTime'
     }
 
     return 'String'
   }
 }
 
+function isValidDate(dateString) {
```

Comment:
Algo assim não resolveria? No caso, ela verificaria se a string é uma data válida ou não de uma forma mais limpa.

```suggestion
function isValidDate(dateString) {
  const date = new Date(dateString);
  return date instanceof Date && !isNaN(date.getTime());
}
```

Reply:
Não, isso ai não funciona